Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DTO;
+using Kernel;
+using System;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class EditUserCommand : IEditUserCommand
+
+    {
+        public void Execute(UserEmailPassword userEmailPassword)
+        {
+            try
+            {
+                CommonValidations.ValidateEmail(userEmailPassword.Email);
+                editUser(userEmailPassword);
```

Comment:
This command edit only UserEmailPassword data. Ask frontend team  what data needs to be changed